controllers/storagecluster/cephcluster.go

```diff
@@ -269,6 +269,12 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 	// Prevent removal of any RDR optimizations if they are already applied to the existing cluster spec.
 	cephCluster.Spec.Storage.Store = determineOSDStore(cephCluster.Spec.Storage.Store, found.Spec.Storage.Store)
 
+	// Use bluestore-rdr if mirrioring is enabled
+	if sc.Spec.Mirroring.Enabled && !sc.Spec.ExternalStorage.Enable {
+		cephCluster.Spec.Storage.Store.Type = string(rookCephv1.StoreTypeBlueStoreRDR)
```

Comment:
Where is the store type set for green field when the annotation is set? How are these related? I guess the type will just be set to the same value by two different sources?

Reply:
Annotation is used only when the new `CephCluster` CR is being created [here](https://github.com/red-hat-storage/ocs-operator/blob/d10e2c8b6a3c4191fcace37c38cc42d1ac8b9f70/controllers/storagecluster/cephcluster.go#L416). 

`yes-really-update-store` only when updating existing `cephCluster` 

>> I guess the type will just be set to the same value by two different sources?
Yes.